Movies/MoviesMVP/MoviesModule/View/MoviesTableViewController.swift

```diff
@@ -0,0 +1,72 @@
+// MoviesTableViewController.swift
+// Copyright © RM. All rights reserved.
+
+import UIKit
+
+protocol MoviesViewProtocol: AnyObject {
```

Comment:
```suggestion
protocol MoviesViewProtocol: AnyObject {
```
лучше держать это в презентере